spec/features/items/index_spec.rb

```diff
@@ -46,16 +53,121 @@
         expect(page).to have_link(@brian.name)
         expect(page).to have_css("img[src*='#{@pull_toy.image}']")
       end
+      expect(page).to have_no_css("#item-#{@dog_bone.id}")
+    end
 
-      within "#item-#{@dog_bone.id}" do
-        expect(page).to have_link(@dog_bone.name)
-        expect(page).to have_content(@dog_bone.description)
-        expect(page).to have_content("Price: $#{@dog_bone.price}")
-        expect(page).to have_content("Inactive")
-        expect(page).to have_content("Inventory: #{@dog_bone.inventory}")
-        expect(page).to have_link(@brian.name)
-        expect(page).to have_css("img[src*='#{@dog_bone.image}']")
+    it "As any kind of user on the system, I can see all items except disabled items" do
+      user = User.create(name: "David", address: "123 Test St", city: "Denver", state: "CO", zip: "80204", email: "123@example.com", password: "password", role: 1)
+      merchant = User.create(name: "Colin", address: "123 Test St", city: "Denver", state: "CO", zip: "80204", email: "456@example.com", password: "password", role: 2)
+      admin = User.create(name: "Max", address: "123 Test St", city: "Denver", state: "CO", zip: "80204", email: "789@example.com", password: "password", role: 3)
```

Comment:
I think it's implied that if a basic user doesn't see something than neither would merchant/visitor/admin but good testing